pyiceberg/table/__init__.py

```diff
@@ -1019,21 +1020,27 @@ def append(self, df: pa.Table) -> None:
         if len(self.spec().fields) > 0:
             raise ValueError("Cannot write to partitioned tables")
 
-        with self.update_snapshot().fast_append() as update_snapshot:
```

Comment:
@Gowthami03B Sorry, this changed quite a bit. I think we now want to pass in the properties through in the `update_snapshot(snapshot_properties=snapshot_properties)` method

Reply:
Hey @Fokko , I am so sorry, have been away for a bit, I am gonna fix this today!

Reply:
@Fokko Done with the changes, how does it look now?